classes/class-loader.php

```diff
@@ -422,6 +422,24 @@ public function enqueue_public_assets() {
 
 		// Sets translated strings for a JS script.
 		wp_set_script_translations( 'planet4-blocks-frontend', 'planet4-blocks', P4GBKS_PLUGIN_DIR . '/languages' );
+
+		if (
+			Features::is_active( Features::THEME_EDITOR_NON_LOGGED_IN )
+			|| ( is_user_logged_in() && Features::is_active( Features::THEME_EDITOR ) )
```

Comment:
I think it's safer to enable the theme editor when both conditions are true, to catch the case where someone accidentally enables the feature on production. Or even to allow that, but just for logged in users.

Reply:
Yeah it's indeed risky. But it's also unfortunate that it has the WP admin bar if you're logged in, which spoils the design a bit. We could also add a check so that it never adds these things on production sites, and maybe not show that checkbox there?

Reply:
You meant checking it like this, right? https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/b18c582f52baa1049bb132516bcbe4bc4b6a4dcb/classes/class-loader.php#L426-L429

Reply:
Yes, but I’m still skeptical about allowing this on productions for non-loggedin users. We can check for environment, but if that’s too complex with our current setup we can wait till we [upgrade](https://make.wordpress.org/core/2020/07/24/new-wp_get_environment_type-function-in-wordpress-5-5/) and just allow it only for logged in users for now.

Reply:
Right, would be too risky, I'll check if we can do an environment check :+1: